L1Trigger/TrackFindingTracklet/plugins/L1FPGATrackProducer.cc

```diff
@@ -739,8 +741,12 @@ void L1FPGATrackProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSe
     aTrack.setStubPtConsistency(
         StubPtConsistency::getConsistency(aTrack, theTrackerGeom, tTopo, settings_.bfield(), settings_.nHelixPar()));
 
+    // set TTTrack word first to allow Track Quality MVA to read variables:
+    aTrack.setTrackWordBits();
+    
     if (trackQuality_) {
       trackQualityModel_->setL1TrackQuality(aTrack);
+      trackQualityDispModel_->setL1TrackQuality(aTrack);
```

Comment:
There should be an extended tracking flag here since this should only run for extended tracking. There is a ```extended_``` variable in this file that can be used.

Reply:
Done here [dbf8525](https://github.com/cms-l1t-offline/cmssw/pull/1257/commits/dbf852576508f471657cead5177e4066460d8381)